src/leader_scheduler.rs

```diff
@@ -322,6 +335,77 @@ impl LeaderScheduler {
     }
 }
 
+pub fn is_leader_rotation_height(
+    height: u64,
+    bootstrap_height: u64,
+    leader_rotation_interval: u64,
+) -> bool {
+    if height < bootstrap_height {
+        return false;
+    }
+
+    if (height - bootstrap_height) % leader_rotation_interval == 0 {
```

Comment:
Redundant if-else-return-false. Reduce to: `(height - bootstrap_height) % leader_rotation_interval == 0`